### ISSUE TITLE
Feature request: Tap on tab to scroll to top (and focus on search field) in all tabs

### ISSUE THREAD

**Issue by boscojwho** [Dimillian/IceCubesApp#1588]
1. Currently, tapping on any tab causes that navigation stack to pop to root, but scroll to top is only implemented on Timeline tab.
2. Additionally, tapping on tab to focus on search field (if already scrolled to top) would be nice.

@Dimillian I wouldn't mind implementing this, unless there's a reason why this was omitted in the first place?

**Comment by Dimillian** [comment]
Yes, it's only the home tab because forwarding it to all tabs is a pain. If you find a good way to do it, feel free!

**Comment by boscojwho** [comment]
Ah fair, let me see what the laziest implementation looks like first.
- I was going to just use the same pattern as the home tab, but that would require injecting the same binding multiple times I suppose 🤨

**Comment by Dimillian** [comment]
Well, it's fine to do it this way, it's ugly but we got to work with what we have 😆

**Comment by boscojwho** [comment]
Got it working on all tabs: 
- Need to fix #1592 first before creating a PR for this.
